src/main/java/gololang/Predefined.java

```diff
@@ -598,7 +603,7 @@ public static boolean isArray(Object object) {
    * @return the Character value.
    * @throws IllegalArgumentException if {@code obj} is not a number or a String.
    */
-  public static Object charValue(Object obj) throws IllegalArgumentException {
+  public static Object charValue(Object obj) {
```

Comment:
Yes, I value the explicitness for unchecked exceptions that may still likely be thrown. Javadoc is one thing, yet `throws` does, too.

The exception is probably `RuntimeException` which is very generic, but all of that is me :smile: